README.md

```diff
@@ -77,19 +77,24 @@ The diagram below describes how the application accesses the configuration.
 
 ## Testing
 
-The tests rely on the test environment set up by controller-runtime.  This
-enables the operator to get initialized against a control pane just like it
-would against a real Openshift cluster.
+The tests rely on the test environment set up by controller-runtime.  This enables the operator to 
+get initialized against a control pane just like it would against a real Openshift cluster.
+
+1. Download and install `kubebuilder`.
+[See install instructions here](https://book.kubebuilder.io/quick-start.html#installation).
+2. Make sure that `kubebuilder` is present in your `PATH`. It is recommended you append it to your
```

Comment:
Just putting the kubebuilder binary in my PATH was not sufficient.  You also need to have `etcd` and `kube-apiserver` binaries available, and afaict you have to specify where they are using the KUBEBUILDER_ASSETS env var.

Reply:
True, when I installed kubebuilder, those two were included. Once they are on the PATH it appears setting the ASSETS is not needed for me.

Reply:
Updated to mention the other executables

Reply:
As long as kubebuilder is installed in the default location (/usr/local/kubebuilder/bin) you don't need KUBEBUILDER_ASSETS ... if you place those in a different dir, you do need that env var. Updated README.